include/llbuild/Basic/ShellUtility.h

```diff
@@ -0,0 +1,43 @@
+//===- ShellUtility.h -------------------------------------------*- C++ -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2015 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef LLBUILD_BASIC_SHELL_UTILITY_H
+#define LLBUILD_BASIC_SHELL_UTILITY_H
```

Comment:
This should be `..SHELLUTILITY...` just to exactly match the file name, but it doesn't really matter and can be fixed up directly on master.